pages/index.tsx

```diff
@@ -33,14 +33,56 @@ const Index = () => {
   if (viewer) {
     return (
       <div>
-        You're signed in as {viewer.email} goto{' '}
-        <Link href="/about">
-          <a>about</a>
-        </Link>{' '}
-        page. or{' '}
-        <Link href="/signout">
-          <a>signout</a>
-        </Link>
+        <div>
+          You're signed in as {viewer.email} goto{' '}
+          <Link href="/about">
+            <a>about</a>
+          </Link>{' '}
+              page. or{' '}
+          <Link href="/signout">
+            <a>signout</a>
+          </Link>
+        </div>
+        <div id="npcApp">
+          <div>
+            <h2>My saved NPCs</h2>
+            <div id="npcListOutput">
+              <p>You have no saved NPCs yet</p>
+              <a hre="#">&#36;&#123;npc.name	&#125;</a>
+            </div>
+            <div id="randomBtnHolder">
+              <button>
+                Roll a random NPC
+              </button>
+            </div>
+          </div>
+          <div>
+            <div id="npcCardTarget">
+
+              <div>
+                <em>&#36;&#123;npc.name	&#125;</em><br />
```

Comment:
This looks like your code got munged by an over-enthusiastic html linter.

Reply:
Fixed.